pkg/types/azure/machinepool.go

```diff
@@ -30,6 +30,13 @@ type MachinePool struct {
 	//
 	// +optional
 	UltraSSDCapability string `json:"ultraSSDCapability,omitempty"`
+
+	// AcceleratedNetworking specifies whether to enable accelerated networking.
+	// Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, greatly improving its
+	// networking performance.
+	//
+	// +optional
+	AcceleratedNetworking string `json:"acceleratedNetworking,omitempty"`
```

Comment:
What are the expected values of this string field, these should be in the go doc

Reply:
I believe Matthew suggested `accelerated` and `basic`. These are what I've tested with. I'll add them to the doc.

Reply:
You should use a type with constant values. For example, see https://github.com/openshift/installer/blob/f6a1b6e3fa71e7165f6e8413215a36e3160182ec/pkg/types/installconfig.go#L376-L391.